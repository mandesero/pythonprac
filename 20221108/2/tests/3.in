(1,1), (2,2), (3,3)
qwe(1,1), (2,2), (3,3)
(1,1), (2,2), (3,3)
(1,1), (2,2), (3,222)