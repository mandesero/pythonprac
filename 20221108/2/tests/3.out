Not a triangle
Invalid input
Not a triangle
109.50
